.github/workflows/prerelease.yml

```diff
@@ -174,3 +174,37 @@ jobs:
           slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
           slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
           slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
+
+  publish-to-s3:
+    name: Send release assets to S3
+    runs-on: ubuntu-20.04
+    needs: [package-win]
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@main
+        env:
+          AWS_S3_BUCKET: <replace-with-real-bucket>
+          AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+          GPG_PRIVATE_KEY_BASE64: ${{ secrets.GPG_PRIVATE_KEY_BASE64 }}
+        with:
```

Comment:
Looks like you are missing this param: `env`
https://github.com/newrelic/infrastructure-publish-action/blob/main/action.yml#L28

Reply:
no tsure yet how we are going to differentiate between re-release and release, if the event we have is `pre-released`

Reply:
we should have 2 of this jobs, on in pre-release pipeline (workflow) another in release pipeline

Reply:
then we'll have to add another workflow for each integration because right now we don't have a release workflow :(